keyboards/satan/satan.h

```diff
@@ -36,6 +36,35 @@
     {k40, k41, k42, XXX, XXX, k45, XXX, XXX, XXX, k49, k4a, k4b, k4c, k4d}  \
 }
 
+
+/* Satan GH60 ANSI layout-B
+ * ,-----------------------------------------------------------.
+ * | 00| 01| 02| 03| 04| 05| 06| 07| 08| 09| 0a| 0b| 0c|    0d |
+ * |-----------------------------------------------------------|
+ * | 10  | 11| 12| 13| 14| 15| 16| 17| 18| 19| 1a| 1b| 1c|  1d |
+ * |-----------------------------------------------------------|
+ * | 20   | 21| 22| 23| 24| 25| 26| 27| 28| 29| 2a| 2b|     2d |
+ * |-----------------------------------------------------------|
+ * | 30     | 32| 33| 34| 35| 36| 37| 38| 39| 3a| 3b|  3d | 3c |
+ * |-----------------------------------------------------------|
+ * | 40 | 41 | 42 |        45              | 4a | 4b | 4c | 4d |
+ * `-----------------------------------------------------------'
+ */
+#define LAYOUT_60_ansi_b( \
```

Comment:
@noroadsleft or @mechmerlin, does this look okay?

Reply:
Looks fine, but I'd prefer the name `LAYOUT_60_ansi_split_rshift` for this, as it's more descriptive.

Reply:
I'd prefer to stick with the existing `LAYOUT_60_ansi_split_bs_rshift` as it's a one key difference in the layout.